src/main/java/it/pagopa/interop/probing/probingapi/service/impl/EserviceServiceImpl.java

```diff
@@ -20,51 +20,52 @@
 @Service
 public class EserviceServiceImpl implements EserviceService {
 
-	@Autowired
-	private OperationsClient operationsClient;
-
-	@Override
-	public void updateEserviceState(UUID eserviceId, UUID versionId,
-			ChangeEserviceStateRequest changeEserviceStateRequest) throws EserviceNotFoundException {
-		operationsClient.updateEserviceState(eserviceId, versionId, changeEserviceStateRequest);
-		log.info("EserviceState of eservice " + eserviceId + " with version " + versionId + " has been updated into "
-				+ changeEserviceStateRequest.geteServiceState());
-	}
-
-	@Override
-	public void updateEserviceProbingState(UUID eserviceId, UUID versionId,
-			ChangeProbingStateRequest changeProbingStateRequest) throws EserviceNotFoundException {
-		operationsClient.updateEserviceProbingState(eserviceId, versionId, changeProbingStateRequest);
-		log.info("EserviceProbingState of eservice " + eserviceId + " with version " + versionId
-				+ " has been updated into " + changeProbingStateRequest.getProbingEnabled());
-	}
+  @Autowired
+  private EserviceClient operationsClient;
```

Comment:
is this on purpose or it should be
```suggestion
  private EserviceClient eserviceClient;
```
?

Reply:
right